doc/userguide/firewall/firewall-design.rst

```diff
@@ -0,0 +1,158 @@
+Firewall Mode Design
+====================
+
+.. note:: In Suricata 8 the firewall mode is experimental and subject to change.
+
+The firewall mode in Suricata allows the use of a ruleset that has different
+properties than the default "threat detection" rulesets:
+
+1. default policy is ``drop``, meaning a firewall ruleset needs to specify what
+   is allowed
+2. firewall rules are loaded from separate files
+3. firewall rules use a new action ``accept``
+4. firewall rules are required to use explicit action scopes and rule hooks (see below)
+
+
+
+Actions and Action Scopes
+-------------------------
+
+Firewall rules require action scopes to be explicitly specified.
+
+accept
+~~~~~~
+
+``accept`` is used to issue an accept verdict to the packet, flow or hook.
```

Comment:
This one is only available in firewall more, right? If so, I think it's best that we indicate such, maybe like:
```suggestion
accept
~~~~~~

``accept`` is used to issue an accept verdict to the packet, flow, or hook. It is
exclusive to firewall mode.
```